.sfdevrc.json

```diff
@@ -1,5 +1,3 @@
 {
-  "test": {
-    "testsPath": "test/**/*.test.ts"
```

Comment:
TODO: re-enable the tests scripts once I add NUTs.
I skipped this as I'll need to set up some hub credentials for this repo.